static/app/components/acl/featureDisabled.tsx

```diff
@@ -70,30 +70,20 @@ class FeatureDisabled extends React.Component<Props, State> {
     this.setState(state => ({showHelp: !state.showHelp}));
   };
 
-  renderFeatureDisabled() {
+  renderContent() {
```

Comment:
Want to convert this into a FC while you're in here 😏

Reply:
sure thing!